functions/exercise14/script.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+function getAmountOfCoins(amount) {
+    var denominationsOfCoins = [25, 10, 5, 2, 1];
+    var amountOfCoins = [];
+    for (var i = 0; i < denominationsOfCoins.length; i++) {
+        if (amount >= denominationsOfCoins[i]) {
+            amountOfCoins.push(denominationsOfCoins[i]);
+            amount -= denominationsOfCoins[i];
+            i = -1;
+            continue;
```

Comment:
It is not the best idea to use break or continue as it is harder to undertand than standard flow. Apart from that is it required here at the end of the loop body? Does it change the behaviour?